test/android/findAndroidAppFolder.spec.js

```diff
@@ -1,12 +1,11 @@
-const path = require('path');
-const expect = require('chai').expect;
+jest.autoMockOff();
+
 const findAndroidAppFolder = require('../../src/config/android/findAndroidAppFolder');
 const mockFs = require('mock-fs');
 const mocks = require('../fixtures/android');
 
 describe('android::findAndroidAppFolder', () => {
-
-  before(() => mockFs({
+  beforeEach(() => mockFs({
```

Comment:
Same here and in other suites, I think that could potentially slow down the test suite (all our functions are just performing reads so there's no need to re-mock every time)